client/src/ledger_app_clients/ethereum/eip712/InputData.py

```diff
@@ -332,7 +397,10 @@ def next_timeout(_signum: int, _frame):
 
 
 def enable_autonext():
-    delay = 1/5
+    if app_client._client.firmware.device in ("stax", "flex"):
+        delay = 1/3
+    else:
+        delay = 1/4
```

Comment:
Questions: did you notice some errors?
Do you have any mean to measure what are the correct values?

Reply:
Yes, it was fine for running the tests but would fail to generate the snapshots correctly on a golden run.
It would skip some indexes : 
* 000.png
* 003.png
* 004.png
* 006.png